frontend/add-leads/add-edit-materialEntry.component.tsx

```diff
@@ -0,0 +1,70 @@
+import { useCss } from "kremling";
+import React from "react";
+import css from "./add-edit-materialEntry.css";
+
+export default function MaterialEntry({
```

Comment:
The name of the component here should match the file name:

```suggestion
export default function AddEditMaterialEntry({
```